### PR TITLE
feat(bigtable): support update column family's value type to non-aggregate type

### PR DESCRIPTION
**Pull request by hoangpham95**

### CODE REVIEW

File: bigtable/admin.go [hunk 1]
```diff
@@ -707,13 +707,38 @@ func (ac *AdminClient) setGCPolicy(ctx context.Context, table, family string, po
 	return err
 }
 
+func (ac *AdminClient) setValueTypeImpl(ctx context.Context, table, family string, valueType Type) error {
+	if _, ok := valueType.proto().GetKind().(*btapb.Type_AggregateType); ok {
```
Comment (bhshkh) [inline bigtable/admin.go:711]: Check valueType nil

**Comment by bhshkh** [comment]
Add integration test

File: bigtable/admin.go (hunk 1 above)
Comment (hoangpham95) [reply bigtable/admin.go:711]: is it needed? valueType is `Type` and not pointer to `Type` so it should always be non-null, right?

**Comment by hoangpham95** [comment]
> Add integration test
Done:
```
go test -test.run="TestIntegration_TestUpdateColumnFamilyValueType" -v \
> -it.use-prod \
> -it.admin-endpoint="test-bigtableadmin.sandbox.googleapis.com:443" \
> -it.data-endpoint="test-bigtable.sandbox.googleapis.com:443" \
> -it.project="google.com:cloud-bigtable-dev" \
> -it.cluster="phhoang-btql-test-c1" \
> -it.instance="phhoang-btql-test"
```

```
Note: when using prod, you must first create an instance:
cbt createinstance phhoang-btql-test phhoang-btql-test phhoang-btql-test-c1 us-central1-b 1 SSD
=== RUN   TestIntegration_TestUpdateColumnFamilyValueType
--- PASS: TestIntegration_TestUpdateColumnFamilyValueType (6.92s)
PASS

ok      cloud.google.com/go/bigtable    59.654s
```

File: bigtable/admin.go (hunk 1 above)
Comment (bhshkh) [reply bigtable/admin.go:711]: In the test, i passed nil
```
err := c.SetValueType(context.Background(), "My-table", "cf1", nil)
		if err != nil && !test.hasError {
			t.Fatalf("Unexpected error when setting type: %v", err)
		}
```

and I don't get any compile time error but see a panic:
```
--- FAIL: TestTableAdmin_SetType (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x10d7942]

goroutine 10 [running]:
testing.tRunner.func1.2({0x129d780, 0x2139e20})
	/usr/lib/google-golang/src/testing/testing.go:1631 +0x24a
testing.tRunner.func1()
	/usr/lib/google-golang/src/testing/testing.go:1634 +0x377
panic({0x129d780?, 0x2139e20?})
	/usr/lib/google-golang/src/runtime/panic.go:759 +0x132
cloud.google.com/go/bigtable.(*AdminClient).setValueTypeImpl(0xc000138c80, {0x16876d0, 0x21a42c0}, {0x14e794e, 0x8}, {0x14e3557, 0x3}, {0x0, 0x0})
	/usr/local/google/home/bahaaiman/Documents/cfdb-workspace-01/hoang/google-cloud-go/bigtable/admin.go:711 +0x62
cloud.google.com/go/bigtable.(*AdminClient).SetValueType(...)
	/usr/local/google/home/bahaaiman/Documents/cfdb-workspace-01/hoang/google-cloud-go/bigtable/admin.go:739
cloud.google.com/go/bigtable.TestTableAdmin_SetType(0xc000124820)
	/usr/local/google/home/bahaaiman/Documents/cfdb-workspace-01/hoang/google-cloud-go/bigtable/admin_test.go:433 +0x205
testing.tRunner(0xc000124820, 0x1556ea0)
	/usr/lib/google-golang/src/testing/testing.go:1689 +0xfb
created by testing.(*T).Run in goroutine 1
	/usr/lib/google-golang/src/testing/testing.go:1742 +0x390
FAIL	cloud.google.com/go/bigtable	0.156s
FAIL
```

Comment (hoangpham95) [reply bigtable/admin.go:714]: Added a nil check.

**Review by bhshkh — APPROVED** [review]

**Pull request merged**